docs/docs/cmd/todo/task/task-add.md

```diff
@@ -34,8 +34,24 @@ m365 todo task add [options]
 `--reminderDateTime [reminderDateTime]`
 : The date and time for a reminder alert of the task to occur. This should be defined as a valid ISO 8601 string in the UTC time zone.
 
+`--categories [categories]`
+: Comma-separated list of categories associated with the task.
+
+`--completedDateTime [completedDateTime]`
+: The date and time when the task was finished. This should be defined as a valid ISO 8601 string. `2021-12-16T18:28:48.6964197Z`
+
+`--startDateTime [startDateTime]`
+: The date and time when the task is scheduled to start. This should be defined as a valid ISO 8601 string. `2021-12-16T18:28:48.6964197Z`
+
+`--status [status]`
+: Indicates the state or progress of the task. The possible values are: `notStarted`, `inProgress`, `completed`, `waitingOnOthers`, `deferred`.
+
 --8<-- "docs/cmd/_global.md"
 
+## Remarks
+
+When you specify the values for `categories`, the possible values for color are pre-set constants such as `None`, `preset0` and `preset1` or a collor mapped property such as `Red`, `Orange` and `blue`. It is permissible to use distinct names.
```

Comment:
What the remarks mention isn't correct. When I use constants like `Red` or `preset0` in Graph Explorer they are applied as custom labels. 

![afbeelding](https://user-images.githubusercontent.com/38426621/222972905-04081907-05d7-440e-afe5-2b7151d0f817.png)

The values they can use for the colored labels aren't pre-set but they can be changed. Take a look at the following: https://learn.microsoft.com/en-us/graph/api/outlookuser-list-mastercategories?view=graph-rest-1.0&tabs=http

![afbeelding](https://user-images.githubusercontent.com/38426621/222972954-ba1ea55b-626b-42fd-ab2e-79bcf190eb37.png)

The Graph docs about creating a new to-do task item state that you'll have to use the `displayName` in the categories property. https://learn.microsoft.com/en-us/graph/api/todotasklist-post-tasks?view=graph-rest-1.0&tabs=http#request-body

When you use the display name instead, then the proper category label, including the color, is defined to the new to-do task. 

![afbeelding](https://user-images.githubusercontent.com/38426621/222973146-343c512a-9024-4e74-abd9-b852408bad01.png)